src/modules/esl-carousel/core/nav/esl-carousel.nav.utils.ts

```diff
@@ -83,52 +84,52 @@ function splitTarget(target: string): {index: string, type: string} {
 }
 
 /** Parses index value defining its value and type (absolute|relative) */
-function parseIndex(index: string | ESLCarouselNavIndex): {value: number, isRelative: boolean, dir?: ESLCarouselDirection} {
+function parseIndex(index: string | ESLCarouselNavIndex): {value: number, isRelative: boolean, direction?: ESLCarouselDirection} {
   if (typeof index === 'number') return {value: index, isRelative: false};
   index = index.trim();
-  if (index === 'next') return {value: 1, isRelative: true, dir: 'next'};
-  if (index === 'prev') return {value: -1, isRelative: true, dir: 'prev'};
-  if (index[0] === '+') return {value: +index, isRelative: true, dir: 'next'};
-  if (index[0] === '-') return {value: +index, isRelative: true, dir: 'prev'};
+  if (index === 'next') return {value: 1, isRelative: true, direction: 'next'};
+  if (index === 'prev') return {value: -1, isRelative: true, direction: 'prev'};
+  if (index[0] === '+') return {value: +index, isRelative: true, direction: 'next'};
+  if (index[0] === '-') return {value: +index, isRelative: true, direction: 'prev'};
```

Comment:
Wouldn't such a proposal be better for comprehension?
```suggestion
  if (index[0] === '-') return makeIndexConfig{+index, true, 'prev');
```

Reply:
It will be less readable, and more antipattern in such a context (the builder arguments are less obviously named than simple object declaration).
PS: the nav utils are still going to be reworked, in the current PR it is a small step to update the carousel API that I going to present in the nearest separate PRs